PHP/PHP Source.sublime-syntax

```diff
@@ -1931,10 +1931,20 @@ contexts:
     - match: '&(?!\s*\$)'
       scope: punctuation.separator.type.intersection.php
       push:
-        - type-hint-nullable
         - type-hint-simple-type
+        - type-hint-nullable
+    - include: type-hint-body-group
     - include: else-pop
 
+  type-hint-body-group:
```

Comment:
`type-hint-group` should be enough. `-body` is a suffix, which should not be followed by anything else.

Reply:
Sorry for my "wrong" hint. With regards to the plural vs. singular thing, this context should be `type-hint-groups` as it pushes a context onto stack.